frontend/src/scenes/project/Settings/CorrelationConfig.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { teamLogic } from 'scenes/teamLogic'
+import { personPropertiesModel } from '~/models/personPropertiesModel'
+import { Select } from 'antd'
+import { DEFAULT_EXCLUDED_PERSON_PROPERTIES } from 'scenes/funnels/funnelLogic'
+
+export function CorrelationConfig(): JSX.Element {
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const { currentTeam } = useValues(teamLogic)
+    const { personProperties } = useValues(personPropertiesModel)
+
+    const handleChange = (excludedProperties: string[]): void => {
+        updateCurrentTeam({ correlation_config: { excluded_person_property_names: excludedProperties } })
+    }
+
+    return (
+        <div style={{ marginBottom: 16 }}>
+            <div style={{ marginBottom: 8 }}>
+                {currentTeam && (
+                    <>
+                        Excluded person properties:{' '}
+                        <Select
+                            mode="multiple"
+                            allowClear
+                            showSearch
+                            value={
+                                currentTeam.correlation_config?.excluded_person_property_names ||
+                                DEFAULT_EXCLUDED_PERSON_PROPERTIES
```

Comment:
I think if we handle the `DEFAULT_EXCLUDED_PERSON_PROPERTIES` in a selector in teams, then this + the part in funnelLogic can be simplified to use only the selector, and the default logic lives only in one place.

Reply:
Yes the only think I don’t like about this is that team then needs to know about the structure. Maybe can unify without coupling though

Reply:
Hmm, fair point. It can be a `funnelLogic` selector, that is then used here as well - but that feels more roundabout tbh.

Reply:
Yeah seems overkill to have constants as selectors.